doc/source/getting_started/XarrayforArviZ.ipynb

```diff
@@ -12,17 +12,17 @@
    "cell_type": "markdown",
```

Comment:
<p>I feel more natural to mention here that there are many ways to start an inferencedata, link to creating inferencedata and then saying here we'll load one example provided by the library for illustration purposes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1782/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>